ontology/yaml/resources/LIGHTING/entity_types/LGRP.yaml

```diff
@@ -20,4 +20,79 @@ LGRP_DC:
   is_canonical: true
   implements:
     - LGRP
-    - DC
\ No newline at end of file
+    - DC
+
+LGRP_OS_SCZ:
+  id: ""
+  description: "Lighing group control with occupancy and zone scene control"
+  is_canonical: true
+  implements:
+  - LGRP
+  - OS
+  - SCZ
+
+LGRP_OS_LC_VADC:
+  id: "4599945020949135360"
```

Comment:
Did this one exist before? its showing up as new; if it is new, please remove the ID. No new type should have an ID. The system sets this.

Reply:
This applies to lines 34 to 98

Reply:
These canonical types (all are LGRP abstract types) were all erroneously place in the LT.yaml file.  I removed them from LT.yaml and moved in to LGRP.yaml.   Assume this means that the ids are all still valid.

Reply:
if they were simply moved, then yes. move them back and leave the IDs as they were.

Reply:
Move them back to LT.yaml?  Isn't that confusing?  I was looking for LGRP canonical types and stumbled in to them in the LT file.

Reply:
oh sorry i was confused. yes, leave them in the new file with the existing type ids.